README.md

```diff
@@ -176,10 +184,10 @@ model = models.get("model-name", pretrained_weights="pretrained-model-name")
 #### Transfer Learning 
   <table class="tfo-notebook-buttons" align="left">
  <td width="500">  
-  <a target="_blank" href="https://bit.ly/3xzIutb"><img src="./docs/assets/SG_img/colab_logo.png" /> Classification Transfer Learning</a>
+  <a target="_blank" href="https://bit.ly/3xzIutb"><img style="vertical-align:middle" src="./docs/assets/SG_img/colab_logo.png" /> Classification Transfer Learning</a>
```

Comment:
Are all these readme.md changes on purpose?

Reply:
It hells in PyCharm preview to align the icons with respect to the text, but GitHub strips this out anyway so I will remove it